src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -759,21 +767,161 @@ public boolean isKnownDatasource( UUID studyId, String participantId, String dat
                 && participantDevices.get( participantId ).containsKey( datasourceId );
     }
 
+    private Void deleteStudyData(
```

Comment:
I think you want `void` instead of `Void` here and in the other two functions which will let you remove the `return null`. `Void` is an object meant for metaprogramming and representing the `void` type whereas `void` is a keyword indicating that a function returns no value (not even null)

Reply:
I actually had a problem with returning void from a service.  See: https://github.com/openlattice/chronicle-api/pull/26

Reply:
Cool so after looking into it (b/c I rarely write Controller endpoints), team standard is to use `Void` so this is all good!